script/DeploySolverAppGateway.s.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.29;
```

Comment:
is there a reason for taking solidity version down?

Reply:
it was not compiling for me